src/index.js

```diff
@@ -1,3 +1,12 @@
+import express from "express";
+import helmet from "helmet";
 import { getDisplayString } from "./render";
```

Comment:
Remove this since there is a proper server started in this file

Reply:
Wanted to remove it, but a conversation initiater. 👍